tests/conftest.py

```diff
@@ -45,7 +41,7 @@ def _add_option(parser: pytest.Parser, name: str, help: str, choices: Optional[L
     )
 
 
-def _get_option(config: pytest.Config, name: str, default: Optional[str] = None) -> str:
+def _get_option(config: pytest.Config, name: str, default: Optional[str] = None) -> str:  # type: ignore
```

Comment:
Why was this type ignore added?

Reply:
Function not used in file (imported elsewhere)